frontend/src/Components/CatalogCourseCard.tsx

```diff
@@ -92,67 +92,71 @@ export default function CatalogCourseCard({
         );
     });
 
-    if (view == ViewType.List)
-        return (
-            <a
-                className="card bg-base-teal body-small p-6 flex flex-row items-center"
-                href={course.external_url}
-                target="_blank"
-            >
-                <div className="flex flex-col justify-between gap-3">
-                    <div className="flex flex-row gap-3 items-center">
-                        <div onClick={(e) => updateFavorite(e)}>{bookmark}</div>
-                        <h2>{course.program_name}</h2>
-                        <p className="body">|</p>
-                        <p className="body">{course.provider_name}</p>
-                        {programPill}
-                        {outcomePills}
-                    </div>
-                    <p className="body-small h-[1rem] line-clamp-2 overflow-hidden">
-                        {course.description}
-                    </p>
-                </div>
-            </a>
-        );
     return (
-        <div className="card card-compact bg-base-teal overflow-hidden relative">
-            <div
-                className="absolute top-2 right-2"
-                onClick={(e) => updateFavorite(e)}
-            >
-                {bookmark}
-            </div>
-            <a
-                href={course.external_url}
-                target="_blank"
-                rel="noopener noreferrer"
-            >
-                <figure className="h-[124px]">
-                    {coverImage !== '' ? (
-                        <img
-                            src={coverImage}
-                            // TO DO: add in alt text here
-                            alt=""
-                            className="object-contain"
-                        />
-                    ) : (
-                        <div className="bg-teal-1 h-full w-full"></div>
-                    )}
-                </figure>
-                <div className="card-body gap-0.5">
-                    {/* this should be the school or program that offers the course */}
-                    <p className="text-xs">{course.provider_name}</p>
-                    <h3 className="card-title text-sm">
-                        {course.program_name}
-                    </h3>
-                    <p className="body-small line-clamp-2">
-                        {course.description}
-                    </p>
-                    <div className="flex flex-wrap py-1 mt-2 space-y-2">
-                        {programPill} {outcomePills}
+        <>
```

Comment:
I got stuck rebasing a massive, massive change (all the formatting) when i rebased from main. So some stuff was confusing. Let me know if this was the wrong change @calisio

Reply:
shoot, ya sorry about having to rebase that 😬 This should have kept the incoming change- the one that separates out the returns (ie the one that has the`if (view == ViewType.List) return (...)` rather than the ternary operator. It was a linting error that I fixed.

Reply:
Ok I kinda figured that was the case, no worries I will fix it